presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -111,6 +116,7 @@
     private final ConcurrentLazyQueue<HivePartitionMetadata> partitions;
     private final Deque<Iterator<InternalHiveSplit>> fileIterators = new ConcurrentLinkedDeque<>();
     private final boolean schedulerUsesHostAddresses;
+    private final Supplier<HoodieROTablePathFilter> pathFilterSupplier;
```

Comment:
nit: let's call it `hoodiePathFilterSupplier`

Reply:
done